### DISCUSSION TITLE
autounattend.xml not able to load during beginning of Setup

### DISCUSSION THREAD

**Discussion by coffeeandcigars** [memstechtips/UnattendedWinstall#155] [Q&A] [answered]
Using freshly downloaded Win 10 x64 ISO (English) directly from MS. (Win10_22H2_English_x64v1.iso)
Bootdrive made in Rufus (4.5.2180 Portable) as MBR for older System. No other Options selected. USB 2 8gb drive from Sandisk.
Without .xml present the install-setup works as expected.
With autounattend.xml (Standard) in place i get an Error Message saying its unable to read the autounattend file.
"Windows Setup encountered an internal error while loading or searching for an unattend answer file"
Any suggestions?

**Comment by coffeeandcigars** [comment, marked as answer]
In case other have the same issue:
I got the .xml by downloading the file by "right click" "save as..."
This unfortunately somehow breaks the xml
When downloading the complete .zip of the project and using the xml in there it works fine.

**Reply by memstechtips** [reply]
Thanks for providing this info.
